kanidmd_web_ui/src/macros.rs

```diff
@@ -0,0 +1,25 @@
+///! Macros for the web UI
+
+/// Adds a default set of CSS classes to the body element
+#[macro_export]
+macro_rules! add_ui_form_classes {
```

Comment:
should say it's "add_body_ui_form_classes" I think?

Reply:
Met you in the middle with `add_body_form_classes` 😄 - ui is redundant identifier in context